features/admin.core.v1/components/header.tsx

```diff
@@ -90,10 +93,18 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (props: HeaderPro
         (state: AppState) => state.config.deployment.accountApp.tenantQualifiedPath
     );
     const isPrivilegedUser: boolean = useSelector((state: AppState) => state.auth.isPrivilegedUser);
-    const feature: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
+    const organizationFeatureConfig: FeatureAccessConfigInterface =
+        useSelector((state: AppState) => state.config.ui.features.organizations);
+    const gettingStartedFeatureConfig: FeatureAccessConfigInterface =
+        useSelector((state: AppState) => state.config.ui.features.gettingStarted);
     const scopes: string = useSelector((state: AppState) => state.auth.allowedScopes);
     const userOrganizationID: string = useSelector((state: AppState) => state?.organization?.userOrganizationId);
 
+    const hasOrganizationReadPermission: boolean =useRequiredScopes(organizationFeatureConfig?.scopes?.read);
+    const hasGettingStartedFeaturePermission: boolean = useRequiredScopes(
```

Comment:
The console feature scopes are about determining whether the user should be able to view the page or not.

```suggestion
    const hasGettingStartedViewPermission: boolean = useRequiredScopes(
```

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6680/commits/a938e76e6c9fbc179ee78185b96e953c5b2a0f3c